yarn.lock

```diff
@@ -94,27 +94,29 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@kaoto-next/ui@npm:2.0.0-TP2":
-  version: 2.0.0-TP2
-  resolution: "@kaoto-next/ui@npm:2.0.0-TP2"
+"@kaoto-next/ui@npm:2.0.0-TP3":
+  version: 2.0.0-TP3
+  resolution: "@kaoto-next/ui@npm:2.0.0-TP3"
   dependencies:
-    "@kaoto-next/uniforms-patternfly": ^0.6.3
+    "@kaoto-next/uniforms-patternfly": ^0.6.8
     "@kie-tools-core/editor": 0.32.0
     "@kie-tools-core/notifications": 0.32.0
-    "@patternfly/patternfly": 5.2.0
+    "@patternfly/patternfly": 5.2.1
```

Comment:
I think we need to also bump resolutions to be in shape with kaoto-next

https://github.com/KaotoIO/vscode-kaoto/blob/main/package.json#L175

Reply:
done.
It is introducing a lot of other changes in yarn.lock